.gitignore

```diff
@@ -0,0 +1,8 @@
+# Environments
```

Comment:
Why are your env files remote whilst they were supposed to be git ignored,  You committed them before you could add them to the gitignore. Please fix this.